identity-service/src/config.js

```diff
@@ -530,7 +530,7 @@ const config = convict({
   },
   solanaSignerPrivateKey: {
     doc: 'solanaSignerPrivateKey',
-    format: 'string',
+    format: String,
```

Comment:
@cheran-senthil probably want this on the working branch

Reply:
I already added this, thanks! I forgot to push to source